src/parser/youtube/Playlist.ts

```diff
@@ -36,7 +36,7 @@ class Playlist extends Feed<IBrowseResponse> {
     this.info = {
       ...this.page.metadata?.item().as(PlaylistMetadata),
       ...{
-        subtitle: header.subtitle,
+        subtitle: header ? header.subtitle : null,
```

Comment:
The rest of this file uses `header?.` instead
Ex:
can_share: header?.can_share,
can_delete: header?.can_delete,
is_editable: header?.is_editable,
privacy: header?.privacy

Reply:
Using `header?.subtitle` would result in it returning `undefined`, which would change the type of the subtitle property from `Text | null` to `Text | null | undefined`. This could be mitigated by doing `header?.subtitle ?? null`, but due to YouTube.js targetting ES2016, it gets transpiled to the rather verbose: `(_b = header === null || header === void 0 ? void 0 : header.subtitle) !== null && _b !== void 0 ? _b : null,`.

I can change the type of `subtitle` to be `Text | undefined`, if you would prefer that over null, I definitely don't want both `null` and `undefined` being possible?